src/main/java/spark/SparkBase.java

```diff
@@ -358,7 +388,7 @@ protected static void addFilter(String httpMethod, FilterImpl filter) {
                                                    + "'", filter.getAcceptType(), filter);
     }
 
-    private static synchronized void init() {
+    public static synchronized void init() {
```

Comment:
Note this has been made public to allow calling N times the `webSocket` method, and allow to start Spark serving only WebSockets. If no routes are added, users should be able to manually call `init()` after having declared the websocket handlers.